src/types/Story.ts

```diff
@@ -282,6 +283,7 @@ export namespace Story {
 
         referenced_entities: {
             stories: Record<string, OEmbedInfo>;
+            coverages: Record<number, CoverageEntry>;
```

Comment:
I believe `coverage` is correct as it's non-countable word: https://www.britannica.com/dictionary/coverage.
Also, did you know that word of the day is [goulash](https://www.britannica.com/dictionary/eb/word-of-the-day)?

Reply:
Well, shoot, I noticed this late. Unfortunately, coverages is all over backend, so it made sense to follow the pattern. 

Similarly, I wouldn't hesitate to say goulashes.

Reply:
What an absolute mad man.